setup.py

```diff
@@ -46,7 +46,7 @@
     zip_safe=False,
     entry_points={
         'console_scripts': [
-            'borg-import = borg_import.main:main',
+            'borg-import = borg_import.main:below_main',
```

Comment:
well it's physically correct - below_main is literally below main - but I'm mostly referring to the C runtime meaning here, the functions actually running before main.

but with the meanwhile refactorings it isn't needed anymore